src/main/java/spoon/support/util/ModelSet.java

```diff
@@ -84,10 +84,7 @@ public boolean add(T e) {
 
 		// we make sure that the element is always the last put in the set
 		// for being least suprising for client code
```

Comment:
Maybe we can remove these comments as well? I don't know why were they added in the first place. The order is anyway controlled by the comparator.

Reply:
Spoon does a lot of unholy things with sets and relaying on their order. I would leave the comment there, because it helps to understand this madness.